container.te

```diff
@@ -685,6 +685,15 @@ tunable_policy(`container_manage_cgroup',`
 	fs_manage_cgroup_files(container_t)
 ')
 
+tunable_policy(`virt_sandbox_use_fusefs',`
+    fs_manage_fusefs_dirs(container_t)
+    fs_manage_fusefs_files(container_t)
+    fs_manage_fusefs_symlinks(container_t)
+    fs_mount_fusefs(container_t)
```

Comment:
Are we sure about mount and unmount?  Those would need to be for the container runtime, not the container itself I think.